apps/mobile/src/features/Settings/SettingsCollateralItem.tsx

```
@@ -15,35 +18,29 @@ export const SettingsCollateralItem = ({
+  const {wallet} = {
+    wallet: {
+      portfolioPrimaryTokenInfo: {
+        id: '.',
+        nature: PortfolioTokenNature.Primary,
+        type: PortfolioTokenType.FT,
+      } as Portfolio.Token.Info,
+    } as YoroiWallet,
+  }
+  const {amount} = {
+    amount: {
+      quantity: 201023131313313311331,
```

Biome (1.9.4)

[error] This number literal will lose precision at runtime.

The value at runtime will be 201023131313313320000

(lint/correctness/noPrecisionLoss)